unity-client/Assets/Builder/Scripts/Gizmos/DCLBuilderGizmo.cs

```diff
@@ -27,16 +27,24 @@ public abstract class DCLBuilderGizmo : MonoBehaviour
         public abstract void SetSnapFactor(DCLBuilderGizmoManager.SnapInfo snapInfo);
         public abstract void TransformEntity(Transform targetTransform, DCLBuilderGizmoAxis axis, float axisValue);
 
-        public virtual void Initialize(Camera camera)
+
+        public virtual void Initialize(Camera camera,Transform _cameraTransform)
         {
+            if (!initialized) initialScale = transform.localScale;
             initialized = true;
-            relativeScaleRatio = transform.localScale / GetCameraPlaneDistance(camera, transform.position);
+            relativeScaleRatio = transform.localScale / GetCameraPlaneDistance(_cameraTransform, transform.position); 
             builderCamera = camera;
+            cameraTransform = _cameraTransform;
             axisX.SetGizmo(this);
             axisY.SetGizmo(this);
             axisZ.SetGizmo(this);
         }
 
+        [ContextMenu("Force relative scale")]
+        public void ForceRelativeScaleRatio()
+        {
+            relativeScaleRatio = new Vector3(0.06f,0.06f,0.06f);
```

Comment:
I deleted the context menu that was the debug part, but the function is necessary. Since I always set the same distance from the player I keep it constant